contracts/test/vault/vault.fork-test.js

```diff
@@ -152,6 +154,82 @@ forkOnlyDescribe("ForkTest: Vault", function () {
       expect(balancePreMint).to.approxEqualTolerance(balancePostRedeem, 1);
     });
 
+    it("should withdraw from and deposit to strategy", async () => {
```

Comment:
Given that these are dangerous write methods, we should have tests for each that verify that the public cannot call each of these, and that the correct revert message happens.

Reply:
yup great point adding that